### ISSUE TITLE
[JIRA] markdown doesn't save lists to output

### ISSUE THREAD

**Issue by layertwo** [saulpw/visidata#406]
**Description**
I have a few files with lists or dictionaries I haven't expanded (they're str.split() examples). When saving to `.jira` filetype, those don't get included in the output file.

When saving to `json`, the output is correct as expected for a JSON doc. When saving to `tsv` the unexpanded list is correctly written as a Python list.

**Additional context**
Running v2.-1 (might be on some weird develop commit).

**Comment by layertwo** [comment]
Since I wrote that saver, I can take a look and try to resolve.

**Comment by saulpw** [comment]
Thanks for both the bug report and for looking into it, @layertwo!  Godspeed and please feel free to ask any questions you have.

**Comment by saulpw** [comment]
Any progress on this, @layertwo?

**Comment by layertwo** [comment]
Sorry for the delay, yeah, I figured out what was wrong. I'll get your a PR this week.

**Comment by layertwo** [comment]
Ok, I've looked at this again. The last time I reviewed (it was a few months ago), there was definitely an issue with how the markdown loader saves unexpanded lists or dictionaries, however I'm not seeing that behavior anymore running the latest commit from develop.

There may be improvements to the markdown loader in the future, but nothing that should hold up 2.0. I'll go ahead and close this issue.
